lib/soapy_bing/accounts.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module SoapyBing
+  class Accounts
+    attr_reader :oauth_credentials, :customer
+
+    def initialize(oauth: {}, customer: {})
+      @oauth_credentials = OauthCredentials.new(oauth)
+      @customer = Customer.new(customer)
+    end
+
+    def list
+      do_request(Soap::Request::GetAccountsInfoRequest).map do |account|
+        Account.new(
+          developer_token: customer.developer_token,
+          customer_id: customer.customer_id,
```

Comment:
Yes, they will be pulled from `ENV` if these params are not provided. 

I follow the same pattern in `SoapyBing::Customer`. But once `SoapyBing::Accounts` is initialized (either by default customer values from ENV or via explicit params in constructor) it should pass the same credentials to build the accounts in `#list`.

What I was wondering is that if I should refactor `SoapyBing::Account` to reference a `SoapyBing::Customer` in place of `developer_token` and `customer_id`.